### ISSUE TITLE
balancing weights for loss 

### ISSUE THREAD

**Issue by Chenzhaowei13** [MedChaabane/DEFT#19]
Hi, thanks for your amazing work.
In the paper, you mentioned using lambda_1 and lambda_2, which  balancing weights to the two tasks.
However,  I found self.s_det and self.s_id are not defined in repo.
```
loss_stats["tot"] = torch.exp(-self.s_det)  * loss_stats["tot"] + torch.exp(-self.s_id) * loss_matching + (self.s_det + self.s_id)
```
Is it convenient to tell the value of the lambda_1 and lambda_2?

**Comment by GondorFu** [comment]
> Hi, thanks for your amazing work. In the paper, you mentioned using lambda_1 and lambda_2, which balancing weights to the two tasks. However, I found self.s_det and self.s_id are not defined in repo.
> 
> ```
> loss_stats["tot"] = torch.exp(-self.s_det)  * loss_stats["tot"] + torch.exp(-self.s_id) * loss_matching + (self.s_det + self.s_id)
> ```
> 
> Is it convenient to tell the value of the lambda_1 and lambda_2?

these two parameters seems defined in ModleWithLoss.loss with 1 and 1
